src/components/DepositWidget/Form.tsx

```diff
@@ -9,7 +9,7 @@ import { formatAmountFull, parseAmount, abbreviateString } from 'utils'
 
 // Components
 import { CardDrawer } from 'components/Layout/Card'
-import { Spinner } from 'components/Spinner'
```

Comment:
shouldnt these then also be called `common/...`

Reply:
Yes, I'm going little by little. The change is big, the ones I'm chaging mostly are related to the Trade widget that is the big monster in the app :) 

Within the Trade Widget, I'ms starting with the price components